native/.eslintrc.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  root: true,
+  extends: ["universe/native"],
```

Comment:
Do we need to add prettier in here?

Reply:
According to the [setup instructions](https://docs.expo.dev/guides/using-eslint/#setup), doesn't looks like it.